osm-seed/values.yaml

```diff
@@ -767,7 +767,7 @@ osmchaApi:
   enabled: false
   image:
     name: "ghcr.io/willemarcel/osmcha-django"
-    tag: "10d7f6748d9e55d439339ed2c35509db15f782a6"
+    tag: "bb038f164f174783775927d14ccc63366f9f2761"
```

Comment:
I think you should use the new ghcr url and tag: https://github.com/OSMCha/osmcha-django/pkgs/container/osmcha-django